airbyte-integrations/connectors/source-mssql/src/test-integration/java/io/airbyte/integrations/source/mssql/CdcMssqlSourceDatatypeTest.java

```diff
@@ -449,17 +444,14 @@ protected void initTests() {
             .createTablePatternSql(CREATE_TABLE_SQL)
             .build());
 
-    // TODO BUG: Airbyte returns binary representation instead of text one.
     // Proper select query example: SELECT test_column.STAsText() from dbo_1_geometry;
     addDataTypeTestData(
         TestDataHolder.builder()
             .sourceType("geometry")
             .airbyteType(JsonSchemaType.STRING)
-            .addInsertValues("null")
-            .addNullExpectedValue()
-            // .addInsertValues("geometry::STGeomFromText('LINESTRING (100 100, 20 180, 180 180)', 0)")
-            // .addExpectedValues("LINESTRING (100 100, 20 180, 180 180)",
-            // "POLYGON ((0 0, 150 0, 150 150, 0 150, 0 0)", null)
+            .addInsertValues("geometry::STGeomFromText('LINESTRING (100 100, 20 180, 180 180)', 0)",
+                "null")
+            .addExpectedValues("LINESTRING(100 100, 20 180, 180 180)", null)
```

Comment:
why do we expect `null` in place of `0` is this the correct behavior?

Reply:
![Selection_359](https://user-images.githubusercontent.com/4688354/178663921-05bc79b8-aa1c-412d-836b-659977fc8db9.png)
We put null and expect null. There are 2 test cases:
1. "geometry::STGeomFromText('LINESTRING (100 100, 20 180, 180 180)', 0)"
2. null